SCons/Tool/msvc.py

```diff
@@ -236,7 +256,7 @@ def generate(env):
         shared_obj.add_emitter(suffix, shared_object_emitter)
 
     env['CCPDBFLAGS'] = SCons.Util.CLVar(['${(PDB and "/Z7") or ""}'])
-    env['CCPCHFLAGS'] = SCons.Util.CLVar(['${(PCH and "/Yu%s \\\"/Fp%s\\\""%(PCHSTOP or "",File(PCH))) or ""}'])
+    env['CCPCHFLAGS'] = CCPCHFLAGS_gen
```

Comment:
This should probably be made into a generator variable like:

```
env['CCPCHFLAGS_GEN'] = CCPCHFLAGS_gen
env['CCPCHFLAGS'] = SCons.Util.CLVar("$CCPCHFLAGS_GEN")
```

to support people appending strings and flags to it?

Reply:
done: 5eb908a